ajax/libs/fg-snapper/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "fg-snapper",
+  "filename": "snapper.min.js",
+  "version": "1.2.1",
+  "description": "A CSS Snap-Points based carousel (and lightweight polyfill)",
+  "author": {
+    "name": "Scott Jehl, Filament Group, Inc.",
+    "url": "https://filamentgroup.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/filamentgroup/snapper.git"
+  },
+  "keywords": [
+    "CSS",
+    "snap",
+    "points",
+    "carousel"
+  ],
+  "license": "MIT",
+  "npmName": "fg-snapper",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
"basePath" should use `dist`
I think when you finished adding old versions of fg-snapper, you forget to revert the package.json.